gxf_extensions/utils/cuda_stream_handler.hpp

```diff
@@ -1,267 +0,0 @@
-/*
```

Comment:
Could you please comment on why this file was removed?

Reply:
Thanks @tbirdso for pointing it out!

I was under the impression that `utils/cuda_stream_handler.hpp` was the same as `include/holoscan/utils/cuda_stream_handler.hpp`, but that wasn't the case.
I reverted this change to ensure the lstm_tensor_rt_inference Operator works properly.

Reply:
Thanks!